config/database.yml

```diff
@@ -43,24 +43,23 @@ test:
     <<: *default
     host: 127.0.0.1
```

Comment:
I changed the host from '127.0.0.1' to 'localhost' in my last ticket that was merged.